google/cloud/security/common/gcp_api/cloud_logging.py

```diff
@@ -0,0 +1,443 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Logging API Client.
+
+Most of this code has been lifted from:
+
+https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/logging/google/cloud/logging
+"""
+
+import json
+import logging
+
+from google.cloud.security.common.gcp_api._base_client import _BaseClient
+# pylint: disable=line-too-long
```

Comment:
You need to re-enable this check after the imports to catch the other places in this file where line is too long.